trading_funcs.py

```diff
@@ -35,11 +40,20 @@
 
 def create_sequences(data: np.array, num_days: int) -> Tuple[np.array, np.array]:
     """
-    Goes into model during fitting to show input and output
-
-    Sequences are input
-    Labels are output
+    The purpose of this function is to create sequences and labels which are implemented
+    into the model during fitting. This is done by iterating through the data and appending
+    the data to the sequences and labels list.
+
+    Args:
+        data (np.array): The data which is used to create the sequences and labels
+        num_days (int): The number of days which is used to create the sequences
+
+    Returns:
+        Tuple[np.array[sequences (list)], np.array[labels (list)]]: The sequences and labels which are used in the model
```

Comment:
Returns:
     tuple: A tuple containing two NumPy arrays.
            - sequences (np.ndarray): An array representing the input of the model
            - label (np.ndarray): An array representing the output of the model